examples/shell-environments/flake.nix

```
@@ -0,0 +1,22 @@
+{
+  description = "Description for the project";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  };
+
+  outputs = inputs@{ flake-parts, ... }:
+    flake-parts.lib.mkFlake { inherit inputs; } {
+      systems = [ "x86_64-linux" "aarch64-darwin" "x86_64-darwin" ];
+      perSystem = { config, self', inputs', pkgs, system, ... }: {
+        devShells.default = pkgs.mkShell {
+          nativeBuildInputs = with pkgs; [ terraform wget bat git ];
```

```suggestion
          nativeBuildInputs = with pkgs; [ terraform wget bat nixpkgs-fmt ];
```

To match the README suggestion earlier.